client/client.go

```diff
@@ -136,11 +142,14 @@ func contains(s []string, e string) bool {
 	return false
 }
 
-func NewClient(host, username, password string) Mikrotik {
+func NewClient(host, username, password string, tls bool, ca_certificate string, insecure bool) Mikrotik {
```

Comment:
ca_certificate => caCertificate to keep consistency with the code base's variable naming conventions.

Reply:
Done